aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -163,6 +163,9 @@ impl<V: Into<Vec<u8>> + Debug + Clone + Eq + Send + Sync + Arbitrary> Transactio
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Arbitrary)]
+pub struct EventType<E: Send + Sync + Debug + Clone>(pub E, pub bool);
```

Comment:
If we are using ContractEvent everywhere, we wouldn't need to make anything generic. Otherwise, since task.rs defines a generic event type - which I think is fine and in line with having the block_executor generic, then here for tests we can have any type that satisfies the type constraints (i.e. seems like Send + Sync + Debug + Clone), and won't need to actually use the type.
So we could have the EventType, but no need to put the E in there? and should also not need to use the ContractEvent anywhere in block executor tests. 

However, the real question is what we need to do to the events themselves, and this would require having some other trait and implementing a mock of this functionality for EventType. That's what KeyType and ValueType do (e.g. we needed TransactionWrite for ValueType). Then based on the mock functionality, we can test whatever block executor does to the provided events.

Reply:
I didn't understand the need of using `EventType`. So, I removed `EventType(E, bool)` and directly used `E` in tests now. Please let me know if I should switch back to using `EventType`.